### DISCUSSION TITLE
unsure how to add a list of options to catogory

### DISCUSSION THREAD

**Discussion by ghost** [Rapptz/discord.py#9728] [General] [answered]
in the slash command menu, i want there to be catogory and i want the options to be ['Support', 'Account', 'Other']

i dont want to use this:

```py
if category not in ['Support', 'Account', 'Other']:
    await ctx.send('Invalid category. Please choose from Support, Account, or Other.')
    return
```
as its not a actual choice menu in discord

code:
```py
import discord
from discord import app_commands

client = discord.Client(intents=discord.Intents.all())
tree = app_commands.CommandTree(client)


@client.event
async def on_ready():
    await tree.sync(guild=discord.Object(id=1198364542334406776))
    print(f'Bot logged in as {client.user} ({client.user.id}). Commands synced.')

@tree.command(name="ticket", description="Create a support ticket", guild=discord.Object(id=1198364542334406776))
async def ticket(interaction, reason: str, catogory: str):
    interaction.response.send_message("This command is not yet implemented.", ephemeral=True)
    

client.run('no')
```

**Comment by Rapptz** [comment, marked as answer]
You can use `typing.Literal` or decorate the parameter with `choices`. [Check the documentation for more information](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.choices).

**Reply by ghost** [reply]
Okay, I will try this.

**Reply by ghost** [reply]
How do I import choice though

**Reply by Rapptz** [reply]
You already have `app_commands` imported. You can use it as `app_commands.choices`. But for your use case I think `typing.Literal` is easier (using `import typing` at the top).

**Reply by ghost** [reply]
Didn't notice I can do app_commands.Choice, thank you
